docs/UserGuide.md

```diff
@@ -198,23 +205,33 @@ Things to add:
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Parameter constraints
+## Patient parameter constraints
 
 _To be cleaned up_
```

Comment:
Would appreciate it if you could help to delete this line since this section should be ok once this PR is merged